finish/src/main/java/io/openliberty/guides/beanvalidation/SerialNumberValidator.java

```diff
@@ -15,22 +15,28 @@
 import javax.validation.ConstraintValidator;
 import javax.validation.ConstraintValidatorContext;
 
+// tag::SerialNumberValidator[]
 public class SerialNumberValidator implements ConstraintValidator<SerialNumber,
                                                                   Object> {
 
     @Override
+    // tag::isValid[]
     public boolean isValid(Object arg0, ConstraintValidatorContext arg1) {
         //Serial Numbers must start with Liberty followed by four numbers
         boolean isValid = false;
         if (arg0 == null)
             return isValid;
         String serialNumber = arg0.toString();
+        // tag::Liberty[]
         isValid = serialNumber.length() == 11 && serialNumber.startsWith("Liberty");
+        // end::Liberty[]
         try {
             Integer.parseInt(serialNumber.substring(7));
         } catch (Exception ex) {
             isValid = false;
         }
         return isValid;
     }
+    // end::isValid[]
 }
+// end::SerialNumberValidator[]
```

Comment:
Add a new line after // end::SerialNumberValidator[]

Reply:
Wouldn't the new line just be extra?

Reply:
It would be but the general practice is to keep the new line so that if someone wants to add the code the cursor by default is at new line.